frontend/public/components/monitoring/metrics.tsx

```diff
@@ -312,7 +312,7 @@ const Query: React.FC<QueryProps> = ({colorOffset, metrics, onBlur, onDelete, on
   const allLabelKeys = _.uniq(_.flatMap(allSeries, s => _.keys(s.labels))).sort();
   const columns = [
     '',
-    ...allLabelKeys.map(title => ({title, ...cellProps})),
+    ...allLabelKeys.map(k => ({title: k === '__name__' ? 'Name' : k, ...cellProps})),
```

Comment:
Are there other internal labels we might want to handle? We could consider calling `_.startCase` on anything with the format `__label__`

https://lodash.com/docs/4.17.15#startCase

Reply:
I think we just have `__name__` currently.

I'm reluctant to add a general rule because it could cause confusion. For example, if `__abc__` is displayed as `Abc`, a user familiar with the labels, may not recognize it as `__abc__`.

I'm just treating `__name__` as a special case because it is so common.